<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCleanForm</class>
 <widget class="QWidget" name="QCleanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_clean_jieshoubei">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_jianguan">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_pengsuan">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_yiqi">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_xiaohuaguan">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_huansuan">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clean_start">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
